zapisy/zapisy/settings.py

```diff
@@ -113,14 +113,17 @@
         'django': {  # configure all of Django's loggers
             'handlers': ['logfile', 'console'] if DEBUG else ['logfile'],
             'level': 'DEBUG',  # set to debug to see e.g. database queries
+            'propagate': False,
         },
         'apps': {
             'handlers': ['logfile'],
             'level': 'DEBUG',
+            'propagate': False,
         },
         'rq.worker': {
             'handlers': ['rq_logfile'],
             'level': 'DEBUG',
+            'propagate': False,
```

Comment:
```suggestion
            'propagate': True,
```
Po namyśle stwierdzam, że dla tego _loggera_ _it's not a bug, it's a feature_.